docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Guide to debugging a Sample Ratio Mismatch (SRM)
+description: What to do if you have a Sample Ratio Mismatch (SRM)
+---
+
+You may be visiting this page from the [Help Center Article](https://help.amplitude.com/hc/en-us/articles/8043418569371-Sample-ratio-mismatches-Debug-issues-with-experiment-allocations) or from the [Blog Post](// TODO (AKHIL) add link here once published). A standard question you may have is what do I do that my experiment has an SRM and I understand what an SRM is.
+
+In this guide, we will give a deeper dive into the process of debugging a SRM. 
+
+!!! info
+    This is mostly for the Experiment product, these same steps work for Experiment Results but we do not have information about the assignment event so it is hard to make some of these graphs. Also, we do not know what the intended traffic allocation is. 
+
+## How to debug an SRM
+
+!!! info
+    If you want a TLDR version of this look at the first five items in the list.
+
+Some of this debugging is similar to debugging general software. Is the SRM only on a certain segment of users (i.e. filter by country, OS version, app version, platform, etc). Was there an instrumentation bug that was pushed to a certain app version?
+
+!!! info
+    This list is sorted in rough priority order. 
+
+### 1. Are you using a custom exposure event or the Amplitude exposure event?
+     
+We recommend you use the Amplitude exposure event as this is sent when `fetch()` is called. (TODO AKHIL Should we say something about local event since that doesn't use Amplitude Exposure). On th other hand, if you use a custom exposure event you may not send it at the time the user is exposed to a variant. Note that it does not cost extra money to use Amplitude Exposure events.
+
+### 2. Did they change traffic allocation in the middle of the experiment?
+
+By traffic allocation, I mean changing 50% treatment / 50% control to 60% treatment / 40% control, not changing it from 2% of all traffic going to the experiment to 5% of all traffic going to the experiment. One of the assumptions of the SRM test is that the traffic allocation does not change in the middle of the test. See this [article](https://help.amplitude.com/hc/en-us/articles/7985566141083-Interpret-the-cumulative-exposures-graph-in-Amplitude-Experiment#inflection-point) for why you should not change the traffic allocation in the middle of an experiment. // TODO AKHIL see if can link to specific paragraph.
```

Comment:
I want to link to this section of the article 

```
It’s strongly recommended that you do not change settings for traffic or traffic allocations to variants in the middle of an experiment; in fact, you will see warnings within Amplitude Experiment to discourage this behavior. Doing so can introduce [Simpson’s paradox](https://en.wikipedia.org/wiki/Simpson's_paradox) into your results (for a more technical explanation and analysis, see [this article](https://www.exp-platform.com/Documents/2009-ExPpitfalls.pdf)). If you have changed the traffic allocation, we suggest you restart the experiment by choosing a new start date. Avoid including any users who were already targeted.
```
But the help center article only lets you link to sections. Is this a big enough thing to ask Spencer about?

Reply:
You can ask, but I dont think you can link to a specific sentence.

Reply:
I checked with Spencer you can only link to a specific section